just/custom.just

```diff
@@ -183,6 +183,11 @@ garden:
         echo ""
     fi
 
+# Install and configure Incus
```

Comment:
The logic to determine whether or not a user needs to rebase to DX could be moved here so that shipping the script on DX only is inconsequential